farmdata2_modules/fd2_tabs/fd2_field_kit/seedingInput.spec.js

```diff
@@ -797,12 +797,18 @@ describe('Test the seeding input page', () => {
                     })
             })
 
-            it('cancel test (alert message and maintain inputs)', () => {
+            it.only('cancel test (alert message and maintain inputs)', () => {
                 cy.get('[data-cy=submit-button]')
                     .click()
                 cy.get('[data-cy=cancel-button]')
                     .click()
-                cy.get('[data-cy=alert-cancel')     // need to add scroll up test here after added
+                    .then(() => {
+                        cy.get('[data-cy=alert-success]')
```

Comment:
The only's are still in there because this isn't meant to be the final commit. For whatever reason, as I detail in my commit message, these only work when I grab them for the opposite banner. So clicking cancel means I have to grab the successful banner (although I can physically verify that banner doesn't appear) and vice versa for clicking confirm.